services/storage/src/simcore_service_storage/db_access_layer.py

```diff
@@ -275,6 +275,15 @@ async def get_file_access_rights(
 # HELPERS -----------------------------------------------
 
 
+async def project_read_access(
+    conn: SAConnection, user_id: UserID, project_id: ProjectID
+) -> bool:
+    access_rights = await get_project_access_rights(
```

Comment:
IMO this extra function is unnecessary ... too many inderections for something that is easy to implement e.g. as
```python
if (project_access = await get_project_access_rights(..)) and project_access.read:
   ...
```

Reply:
yep, good point. I have cleaned this up